index.html

```diff
@@ -8,50 +8,50 @@
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-    <top_nav>
-        <item_nav>
-            <frame4>
-                <portfolio>portfolio</portfolio>
-            </frame4>
-            <frame3>
-                <about>About</about>
-            </frame3>
-            <frame5>
-                <contact>Contact</contact>
-            </frame5>
+    <to1_nav1 id="to1_nav1">
+        <item_nav id="item_nav">
+            <fram4>
+                <portf id="portf">portfolio</portf>
+            </fram4>
+            <fram3>
+                <about id="about">About</about>
+            </fram3>
+            <fram5>
+                <contact id="contact">Contact</contact>
+            </fram5>
         </item_nav>
 
-        <name>
-            <johndoe>John doe</johndoe>
-        </name>
+        <nam id="nam">
+            <johndoe id="johndoe">John doe</johndoe>
+        </nam>
 
-        <rectangle></rectangle>
+        <rectang id="rectang"></rectang>
 
-        <normal_btn>
-            <icons>
-                <union1>
+        <normal_btn id="normal_btn">
+            <icons id="icons">
+                <union1 id="union1">
+                    <vectorx id="vectorx">
                     <vectorx>
-                    <vectorx>
-                    <rec>
+                    <rec id="rec">
                         <img src="./assets/Icon - Mail.png" alt="">
                     </rec>
                 </union1>
             </icons>
         </normal_btn>
 
-        <hamburger>
+        <hamburger id="hamburger">
             <img src="./assets/Enabled.svg" alt="">
         </hamburger>
```

Comment:
- [ ] Kindly I recommend you to use `nav` tag for  hamburger button.The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.